### DISCUSSION TITLE
Statistic Graph Card should allow for averaging data ona user-defined period

### DISCUSSION THREAD

**Discussion by wigster** [home-assistant/frontend#9719] [Dashboard]
Currently, the statistics graph plots the long-term data in a raw fashion: the time series I guess has a granularity of one hour and the max/min/mean is taken on a sample of that length.

However, for plots which are longer than a few days it might make more sense to plot data averaged on e.g. a daily, or even monthly basis. For example 

* min/max temperatures in each 24-h period over a period of 1 year. 
* Energy use per month over 12 months

It would be very convenient if an option existed which would allow the user to set the averaging period, with a minimum of the current one hour. 

The mean/min/max of a set of mean/min/max statistics is the same as that of the original data, so these are not new data series that would have to be stored and I presume are short enough so that they could be calculated in real time form the long-term statistics data series.

**Comment by nsmits** [comment]
#9847 is a possible duplicate. Same question/request.
